### PR TITLE
Refactor GA Singularity

### PR DESCRIPTION
**Pull request by guacke**
Trying to help @dariatols with Singularity permission issues.

### CODE REVIEW

**Comment by guacke** [comment]
Investigating 'Error running Singularity with sudo' on https://sylabs.io/guides/2.5/user-guide/troubleshooting.html.

**Comment by guacke** [comment]
Adapting https://github.com/singularityhub/github-ci/blob/master/.github/workflows/manual-deploy.yml.

**Comment by guacke** [comment]
Currently having problems installing packages through apt-get inside the Singularity sif.

**Comment by guacke** [comment]
GQCP-dev Singularity can be built. @dariatols I have disabled artifacts as we would incur very large storage. We could however push GQCP-dev/GQCP Singularity to [Sylabs.io, the singularity alternative to DockerHub](https://sylabs.io/guides/3.2/user-guide/cloud_library.html#overview) on push to develop. I don't have the impression that Sylabs.io supports organization accounts. @dariatols Can you look into uploading the built .sif files to your account on Sylabs.io?

File: singularity/GQCP-dev.def [hunk 1]
```diff
@@ -29,5 +34,5 @@ From: ubuntu:20.04
     exec echo "$@"
 
 %labels
-    Author GQCP
+    Author GQCG
     Version v0.1.0
```
Comment (dariavh) [inline singularity/GQCP-dev.def]:
```suggestion
    Version v0.1.0
    
```

**Review by dariavh — APPROVED** [review]

**Pull request merged**